osimpipeline/vital_tasks.py

```diff
@@ -870,10 +899,204 @@ def __init__(self, trial, cmc_setup_task, **kwargs):
+
+        self.file_dep += [
+            self.kinematics_file,
+            self.kinetics_file
+        ]
+
+        # Fill out setup.m template and write to results directory
+        self.create_setup_action()
+
+        # Fill out postprocess.m template and write to results directory
+        self.add_action(
+                ['templates/mrs/postprocess.m'],
+                [self.results_post_fpath],
+                self.fill_postprocess_template)
+
+    def create_setup_action(self): 
+        self.add_action(
+                    ['templates/%s/setup.m' % self.tool],
+                    [self.results_setup_fpath],
+                    self.fill_setup_template,  
+                    init_time=self.init_time,
+                    final_time=self.final_time,      
+                    )
+
+    def fill_setup_template(self, file_dep, target,
+                            init_time=None, final_time=None):
+        with open(file_dep[0]) as ft:
+            content = ft.read()
+            content = content.replace('@STUDYNAME@', self.study.name)
+            content = content.replace('@NAME@', self.tricycle.id)
+            # TODO should this be an RRA-adjusted model?
+            content = content.replace('@MODEL@', os.path.relpath(
+                self.subject.scaled_model_fpath, self.path))
+            content = content.replace('@REL_PATH_TO_TOOL@', os.path.relpath(
+                self.optctrlmuscle_path, self.path))
+            # TODO provide slop on either side? start before the cycle_start?
+            # end after the cycle_end?
+            content = content.replace('@INIT_TIME@',
+                    '%.5f' % init_time)
+            content = content.replace('@FINAL_TIME@', 
+                    '%.5f' % final_time)
+            content = content.replace('@IK_SOLUTION@',
+                    self.rel_kinematics_file)
+            content = content.replace('@ID_SOLUTION@',
+                    self.rel_kinetics_file)
+            content = content.replace('@SIDE@',
+                    self.trial.primary_leg)
+
+        with open(target[0], 'w') as f:
+            f.write(content)
+
+    def fill_postprocess_template(self, file_dep, target):
+        with open(file_dep[0]) as ft:
+            content = ft.read()
+            content = content.replace('@STUDYNAME@', self.study.name)
+            content = content.replace('@NAME@', self.tricycle.id)
+            content = content.replace('@REL_PATH_TO_TOOL@', os.path.relpath(
+                self.optctrlmuscle_path, self.path))
+            content = content.replace('@MODEL@', os.path.relpath(
+                self.subject.scaled_model_fpath, self.path))
+
+        with open(target[0], 'w') as f:
+            f.write(content)
+
+class TaskMRSDeGroote(task.ToolTask):
+    REGISTRY = []
+    def __init__(self, trial, mrs_setup_task, **kwargs):
+        kwargs['opensim'] = False
+        super(TaskMRSDeGroote, self).__init__(mrs_setup_task, trial, **kwargs)
```

Comment:
The `kwargs['opensim'] = False` line is fine with me but you can still pass your own keyword arguments even if you have `**kwargs`.